src/main/resources/edu/cornell/kfs/fp/document/validation/configuration/CU-DisbursementVoucherValidation.xml

```diff
@@ -98,4 +98,18 @@
 	</property>
 	</bean>
 	
+	<bean id="RecurringDisbursementVoucher-saveDocumentValidation" parent="DisbursementVoucher-saveDocumentValidation-parentBean">
+		<property name="validations">
+			<list merge="true">
+				<bean parent="RecurringDisbursementVoucherForeignVendorValidation" scope="prototype">
+                    <property name="parameterProperties">
+                        <list>
+                            <bean parent="accountingDocumentFieldConversion"/>
+                        </list>
+                    </property>
+                </bean>
+			</list>

```

Comment:
There's a mix of tabs and spaces in this section. It would be helpful to keep the formatting consistent.